spec/unit/puppet/type/iis_application_pool_spec.rb

```diff
@@ -113,7 +113,6 @@
   
   [
     :name,
-    :clr_config_file,
```

Comment:
Line 131.

Though I've identified a potential issue here: at least one 'path' attribute is documented as accepting variables:

https://docs.microsoft.com/en-us/iis/configuration/system.applicationhost/sites/sitedefaults/logfile/

`directory = %SystemDrive%\inetpub\logs\LogFiles`

I don't think this is true or common with physicalpath, but clr_config_file and source (iis_feature)?